### PR TITLE
fix(nms): statistics not showing for non super users

### LINKED ISSUE #8847: (orc8r) Non-Superuser is not  able to see Grafana charts in NMS
Non Super users are not able to see Grafana charts on the metrics page.
They are still shown the older display of the statistics. 
@karthiksubraveti

### PR DESCRIPTION
**Pull request by paulocalaes**
Signed-off-by: Paulo Henrique Calaes Oliveira <paulo@radtonics.com>

## Summary

This pull request solve the issue https://github.com/magma/magma/issues/8847

Closes #8847

## Test Plan

Open metrics page with non super user and check statistics.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by sebathomas** [comment]
Opening the Grafana page is still not possible for me with your changes.

(I think it's related to the `isSuperUser` check for `grafanaEnabled` in `LteMetrics.tsx`. However, if I try simply removing that I get the Grafana tab but I'm still not able to access it.)

**Comment by paulocalaes** [comment]
@sebathomas strange, I have tested, this problem was related to the Gateway Selector, if you check, is rendering One level Up on the Object.

image

With the fix, we have this behavior:

image

**Comment by sebathomas** [comment]
Ah, I see. I agree it solves that problem and makes it possible to access the gateway metrics. Not sure if it solves the linked issue though, I think the idea there was to give access to Grafana.

File: nms/app/components/insights/GatewayMetrics.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default function (props: {configs: Array<MetricGraphConfig>}) {
     return <LoadingFiller />;
   }
 
-  const gatewayNames = Object.keys(selectors);
+  const gatewayNames = Object.keys((selectors as any).gateways);
```
Comment (sebathomas) [inline nms/app/components/insights/GatewayMetrics.tsx]: Do you still get linter issues if you simply do
```suggestion
  const gatewayNames = Object.keys(selectors.gateways);
```
?

**Review by sebathomas — APPROVED** [review]
Looks good to me, tested that I was able to access the gw information on my machine.

To get the CI green (DCO check), you will probably need to amend the second commit with a sign-off and force push.

**Comment by paulocalaes** [comment]
> Looks good to me, tested that I was able to access the gw information on my machine.
> 
> To get the CI green (DCO check), you will probably need to amend the second commit with a sign-off and force push.

Done!

**Pull request merged**